### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by mixmix**
This is pretty ugly looking, but has minimal functions.

- click on a name and be taken to the profile.
- see conversations you've had with this person, and click into them
- start a new direct conversation with them. (button to page only)

Still needs : 
- style
- better cards
- maybe a bounded stream for private threads.

### CODE REVIEW

**Comment by mixmix** [comment]
@dominictarr I started using the `state.obs` but it was a bit more complicated than building what I've made here quickly

**Comment by mixmix** [comment]
oh yeah. I started to refactor out the `thread-card.mcss` which will make reworking other stuff easier latter

File: app/page/userShow.js [hunk 1]
```diff
@@ -17,20 +27,71 @@ exports.create = (api) => {
...
-      h('div', 'groups dominic is in'),
+      h('h1', api.about.obs.name(feed)),
+      h('div.container', [
+        api.about.html.image(feed),
+        feed !== myId
+          ? h('div.friendship', [
+            h('div.state', ourRelationship),
+            followButton
+          ]) : '',
+        h('div', '...friends in common'),
```
Comment (dominictarr) [inline app/page/userShow.js]: move these strings into translations!

**Comment by dominictarr** [comment]
good call on not using state.obs here, it still needs a thing to work in this case.

File: app/page/userShow.js (hunk 1 above)
Comment (mixmix) [inline app/page/userShow.js]: I'll comment this out

**Pull request merged**